### ISSUE TITLE
Automatically generated Fortran include paths too long

### ISSUE THREAD

**Issue by cniethammer** [open-mpi/ompi#6367]
## Background information
I am trying to build Open MPI 4.0.0 with gcc 8.2.0 under CentOS 7 with the following configure options:
--prefix=/opt/mpi/openmpi/4.0.0-gnu-8.2.0 --enable-shared --enable-static --enable-mpi-thread-multiple
The build directory is a longer path: /<...>/work/mpi/openmpi/4.0.0-gnu-8.2.0/build/

## Details of the problem
The Fortran module creation fails due to a too long include path that seems to be generated automatically. For F90 it is truncated at line 80, which stops the build with
```shell
PPFC     mpi-ext-module.lo
mpi-ext-module.F90:29.6:

      include '/<...>/work/mpi/openmpi/4.0.0-gnu-8.
      1
Error: Unclassifiable statement at (1)
...
```

**Comment by kawashima-fj** [comment]
Thanks for the report.

This issue is a duplicate of #6105 and #6106 and is fixed in #6109. The fix will be included in v4.0.1. Until v4.0.1 is released, you can try [a v4.0.x nightly snapshot tarball](https://www.open-mpi.org/nightly/v4.0.x/) or manually edit following two files in the build directory after running `configure`. Please replace all `include` to `#include` in the files.

```
ompi/mpi/fortran/mpiext-use-mpi/mpi-ext-module.F90
ompi/mpi/fortran/mpiext-use-mpi-f08/mpi-f08-ext-module.F90
```

**Comment by jsquyres** [comment]
Thanks @kawashima-fj.  I'm going to close this issue since you correctly marked it as a duplicate.
